data/exploits/cve-2015-5287/sosreport-rhel7.py

```diff
@@ -48,40 +48,40 @@ def waitpath(p):
 
 dumpdir = waitpath("/var/tmp/abrt/cc*%d" % pid)[0]
 
-print "dump directory: ", dumpdir
+print("dump directory: ", dumpdir)
 
 os.chdir(dumpdir)
 
-print "waiting for sosreport directory"
+print("waiting for sosreport directory")
 
 sosreport = waitpath("sosreport-*")[0]
 
-print "sosreport: ", sosreport
+print("sosreport: ", sosreport)
 
-print "waiting for tmpfiles"
+print("waiting for tmpfiles")
 tmpfiles = waitpath("tmp*")
 
-print "tmpfiles: ", tmpfiles
+print("tmpfiles: ", tmpfiles)
 
-print "moving directory"
+print("moving directory")
 
 os.rename(sosreport, sosreport + ".old")
 os.mkdir(sosreport)
-os.chmod(sosreport,0777)
+os.chmod(sosreport,0o777)
```

Comment:
This is an old exploit that runs only on targets that have Python 2 installed as the default and does not need to be updated.

Reply:
Ok, I'll revert all the changes to this file.